code/reposadolib/reposadocommon.py

```diff
@@ -230,6 +230,24 @@ def print_stderr(msg, *args):
         print >> sys.stderr, concat_message(msg, *args)
 
 
+def humanReadable(size_in_bytes):
+    """Returns sizes in human-readable units."""
+    try:
+        size_in_bytes = int(size_in_bytes)
+    except ValueError:
+        size_in_bytes = 0
+    units = [(" bytes", 2**10, int),
+             (" KiB", 2**20, float),
+             (" MiB", 2**30, float),
+             (" GiB", 2**40, float),
+             (" TiB", 2**50, float)]
+    for suffix, limit, cast_unit in units:
+        if size_in_bytes > limit:
+            continue
+        else:
+            return str(cast_unit(round(size_in_bytes/float(limit/2**10), 1))) + suffix
+
+
```

Comment:
Few things: 1) HumanReadableSizes needs to be explicitly set in your preferences else you get the previous behavior. This means that the only "change" with the KiB/MiB, etc. is in the `repoutil --info` output. I'm not sure how many folks would be parsing `info` output. 2) I really don't feel strongly about it. I actually figured you'd comment on that part :), so that whole commit can be reversed if you want. 3) If that commit were reversed would you be averse to changing the units to proper SI units (multiples of 1000) vs. binary prefixes? Output format wouldn't change, but the numbers would, slightly.